apisupport/apisupport.installer/nbproject/org-netbeans-modules-apisupport-installer.sig

```diff
@@ -159,6 +159,8 @@ meth public java.awt.im.InputMethodRequests getInputMethodRequests()
 meth public java.awt.image.ColorModel getColorModel()
 meth public java.awt.image.VolatileImage createVolatileImage(int,int)
 meth public java.awt.image.VolatileImage createVolatileImage(int,int,java.awt.ImageCapabilities) throws java.awt.AWTException
+meth public java.awt.peer.ComponentPeer getPeer()
```

Comment:
It would be better to remove all these getPeer() methods from the listing. They are not present on newer JDK and if we include them in the `.sig` file, then the `ant build` (which includes sigtest check) will fail on JDK11+.

Reply:
OK, done - please double check @JaroslavTulach 

Used -

```
find . -name "*.sig" -exec sed -i '/java.awt.peer.ComponentPeer/{N;d;}' {} \;
```

If that looks correct we should document that unless/until generation is fixed.  Initially matched on `getPeer()` until I realised there's a non-AWT method with same name.

Reply:
Great:
```
curl https://patch-diff.githubusercontent.com/raw/apache/netbeans/pull/2338.diff | grep getPeer
```
yields nothing. Good job.